packages/engine/stdlib/src/ts/neighbor.spec.ts

```diff
@@ -25,19 +28,25 @@ test("find neighbors with same position", () => {
 });
 
 test("find neighbors within a radius of 3", () => {
-  expect(neighborsInRadius(ng, [na, nb, nc, nd, ne, nf, nh, ni], 3, 0, "chebyshev")).toEqual([
-    { position: [6, 9, 0] },
-    { position: [4, 9, 0] },
-  ]);
+  expect(
+    neighborsInRadius(ng, [na, nb, nc, nd, ne, nf, nh, ni], 3, 0, "chebyshev"),
+  ).toEqual([{ position: [6, 9, 0] }, { position: [4, 9, 0] }]);
 });
 
 test("find neighbors within a max radius of 4 and min radius of 3", () => {
   expect(
-    neighborsInRadius(ng, [na, nb, nc, nd, nf, nh, ni, nj], 4, 3.5)
+    neighborsInRadius(ng, [na, nb, nc, nd, nf, nh, ni, nj], 4, 3.5),
   ).toEqual([ni]);
 
   expect(
-    neighborsInRadius(ng, [na, nb, nc, nd, nf, nh, ni, nj], 7, 3.5, "euclidean", true)
+    neighborsInRadius(
```

Comment:
@litvand Your favourite kind of inconsistent formatting haha

Reply:
Yeah... Maybe `[na, nb, nc, nd, nf, nh, ni, nj]` can be assigned to a variable to make it consistent.